CRVResponse/fcl/prolog_extracted_v01.fcl

```diff
@@ -0,0 +1,16 @@
+#include "CRVResponse/fcl/prolog.fcl"
+
+physics.producers.CrvPhotons.crvStepModuleLabels  : ["CrvSteps"]
+physics.producers.CrvPhotons.crvStepProcessNames  : ["CrvStepsConverter"]
+physics.producers.CrvPhotons.CRVSectors           : ["EX","T1","T2"]
+physics.producers.CrvPhotons.reflectors           : [  0,   1,   0 ]
+physics.producers.CrvPhotons.lookupTableFileNames : [
+                                                     "CRVConditions/v6_0/LookupTable_6000_0",
+                                                     "CRVConditions/v6_0/LookupTable_2370_1",
+                                                     "CRVConditions/v6_0/LookupTable_3200_0",
+                                                    ]
+physics.producers.CrvCoincidence.CRVSectors                   : [ "EX", "T1", "T2"]
+physics.producers.CrvCoincidence.PEthresholds                 : [  20 ,  20,   20 ]
+physics.producers.CrvCoincidence.adjacentPulseTimeDifferences : [  10 ,  10,   10 ]
+physics.producers.CrvCoincidence.maxTimeDifferences           : [  20 ,  20 ,  20 ]
+physics.producers.CrvCoincidence.useFourLayers                : [false, true, true]
```

Comment:
I don't think the absence of a newline is a problem for art, but it would be good to check.

Reply:
I agree, but I'm not sure how to test this as I'm not familiar with how this fcl should be used. I just added this as Ralf gave it to me along with the starting point for the CRV module geometry configuration.

Reply:
I recommend just adding the new line.  Commit and push to your repo.  The PR will pick it up automatically.

Reply:
I've added this line, and pushed the commit.